aws_sqs/lib/botocore/data/storagegateway/2013-06-30/examples-1.json

```diff
@@ -0,0 +1,1381 @@
+        },
+        "output": {
+          "GatewayARN": "arn:aws:storagegateway:us-east-1:111122223333:gateway/sgw-12A3456B"
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "Configures one or more gateway local disks as upload buffer for a specified gateway.",
+        "id": "to-add-upload-buffer-on-local-disk-1471293902847",
+        "title": "To add upload buffer on local disk"
+      }
+    ],
+    "AddWorkingStorage": [
+      {
+        "input": {
+          "DiskIds": [
+            "pci-0000:03:00.0-scsi-0:0:0:0",
+            "pci-0000:03:00.0-scsi-0:0:1:0"
+          ],
+          "GatewayARN": "arn:aws:storagegateway:us-east-1:111122223333:gateway/sgw-12A3456B"
+        },
+        "output": {
+          "GatewayARN": "arn:aws:storagegateway:us-east-1:111122223333:gateway/sgw-12A3456B"
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "Configures one or more gateway local disks as working storage for a gateway. (Working storage is also referred to as upload buffer.)",
+        "id": "to-add-storage-on-local-disk-1471294305401",
+        "title": "To add storage on local disk"
+      }
+    ],
+    "CancelArchival": [
+      {
+        "input": {
+          "GatewayARN": "arn:aws:storagegateway:us-east-1:111122223333:gateway/sgw-12A3456B",
+          "TapeARN": "arn:aws:storagegateway:us-east-1:999999999999:tape/AMZN01A2A4"
+        },
+        "output": {
+          "TapeARN": "arn:aws:storagegateway:us-east-1:999999999999:tape/AMZN01A2A4"
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "Cancels archiving of a virtual tape to the virtual tape shelf (VTS) after the archiving process is initiated.",
+        "id": "to-cancel-virtual-tape-archiving-1471294865203",
+        "title": "To cancel virtual tape archiving"
+      }
+    ],
+    "CancelRetrieval": [
+      {
+        "input": {
+          "GatewayARN": "arn:aws:storagegateway:us-east-1:111122223333:gateway/sgw-12A3456B",
+          "TapeARN": "arn:aws:storagegateway:us-east-1:999999999999:tape/AMZN01A2A4"
+        },
+        "output": {
+          "TapeARN": "arn:aws:storagegateway:us-east-1:999999999999:tape/AMZN01A2A4"
+        },
+        "comments": {
+          "input": {
+          },
+          "output": {
+          }
+        },
+        "description": "Cancels retrieval of a virtual tape from the virtual tape shelf (VTS) to a gateway after the retrieval process is initiated.",
+        "id": "to-cancel-virtual-tape-retrieval-1471295704491",
+        "title": "To cancel virtual tape retrieval"
+      }
+    ],
+    "CreateCachediSCSIVolume": [
+      {
+        "input": {
+          "ClientToken": "cachedvol112233",
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical
<details>
<summary><strong>Description</strong></summary>

A generic secret or password is an authentication token used to access a computer or application and is assigned to a password variable.

</details>


Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- f3a93b18-45f1-4572-9a72-e2974a48103b -->

Reply:
This is an example Token part of the AWS SDK library.